apps/mobile/src/components/Feed/Posts/PostListCaption.tsx

```diff
@@ -32,7 +43,24 @@ export function PostListCaption({ feedPostRef }: Props) {
 
   return (
     <View className="px-4 pb-4">
-      <ProcessedText text={captionWithMarkdownLinks} mentionsRef={nonNullMentions} />
+      <View>
+        <GalleryTouchableOpacity
+          eventElementId="Show more lines"
+          eventName="Show more lines"
+          eventContext={eventContext || null}
```

Comment:
instead of receiving `eventContext` to props of this component, let's just hard-code here:

```js
import { contexts } from '~/shared/analytics/constants';

...

eventContext={contexts.Posts}
```

Reply:
done! normally all the contexts are just hard-code? or should I get it from props normally?